client/modules/core/main.js

```diff
@@ -474,7 +474,7 @@ export default {
   getShopPrefix() {
     const shopName = this.getShopName();
     if (shopName) {
-      return `/${this.getSlug(shopName.toLowerCase())}`;
+      return `/shop/${this.getSlug(shopName.toLowerCase())}`;
```

Comment:
I know that this is not the point of the PR, but a Shop _has_ a slug property. Computing one from the name feels like a bug.
(less important, but `slugify`, the underlying method for `getSlug` lowercases by default)

Reply:
@pmn4 Thanks for pointing that out. I was not sure if I should change that in this PR. 
But maybe I should.